modules/terraform/format.go

```diff
@@ -21,6 +21,14 @@ var TerraformCommandsWithLockSupport = []string{
 	"import",
 }
 
+// SignalValue indicates to the argument formater of some special handling
+type SignalValue int
+
+const (
+	// KeyOnly tells the formatter to omit the value and handle the option as a top-level assignment
+	KeyOnly SignalValue = iota
+)
```

Comment:
I'm a bit confused... Where is this actually used? Is the PR missing some files?

Reply:
I apologize, I only pushed up the failing test. The actual fix was stashed.